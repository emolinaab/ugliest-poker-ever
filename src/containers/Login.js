import React from 'react';

const Login = () => (
  <section></section>
);

export default Login;
