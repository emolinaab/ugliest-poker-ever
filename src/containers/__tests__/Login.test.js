import React from 'react';
import { shallow } from 'enzyme';
import Login from '../Login';

describe('Login Component', () => {
  it('should render correctly', () => {
    const component = shallow(<Login />);
  
    expect(component).toMatchSnapshot();
  });
});
