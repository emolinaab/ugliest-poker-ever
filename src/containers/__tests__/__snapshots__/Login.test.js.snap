// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Component should render correctly 1`] = `<section />`;
